pkg/controllers/disruption/helpers.go

```diff
@@ -55,21 +54,21 @@ func SimulateScheduling(ctx context.Context, kubeClient client.Client, cluster *
 	candidateNames := sets.NewString(lo.Map(candidates, func(t *Candidate, i int) string { return t.Name() })...)
 	nodes := cluster.Nodes()
 	deletingNodes := nodes.Deleting()
-	stateNodes := lo.Filter(nodes.Active(), func(n *state.StateNode, _ int) bool {
+	stateNodes := lo.Filter(nodes.Active().Nodes, func(n *state.StateNode, _ int) bool {
```

Comment:
Why do we have to call `Nodes` again now. This looks pretty awkward to me at first glance

Reply:
Yeah this is because we're wrapping it with kubeClient now. I thought it was cleaner at first, but after having to make these changes, I'm going to revert it to the original.